downstream/modules/eda/proc-eda-duplicate-credential.adoc

```diff
@@ -0,0 +1,21 @@
+[id="eda-duplicate-credential"]
+
+= Copying a credential
+
+When setting up a new credential with field inputs that are similar to your existing credentials, you can use the *Copy credential* feature in the Details tab to duplicate information instead of manually entering it. While setting up credentials can be a lengthy process, the ability to copy the required fields from an existing credential saves time and, in some cases, reduces the possibility of human error.
+
+.Procedure
+
+. On the Credentials list page, click the name of the credential that you want to copy. This takes you to the *Details* tab of the credential.
```

Comment:
We can also note that this can be done directly from the `Credentials list page` by clicking the `Copy credential` button from the list view.

Reply:
Will incorporate this into the information. Thanks, @zkayyali812 .